src/style.css

```diff
@@ -708,4 +708,67 @@ tbody td {
             border-radius: 50px;
```

Comment:
noticing this css file is also almost 800 lines I'd reccomend breaking this up into multiple files

Reply:
I just talked to Jose about this, we'll organize everything in a different issue